### ISSUE TITLE
make a gsi build suitable for android go phones ( like redmi go has only 930 mb system )

### ISSUE THREAD

**Issue by proganime1200** [phhusson/treble_experimentations#519]

**Comment by phhusson** [comment]
My Redmi Go has a 1.3GB system partition...?

**Comment by proganime1200** [comment]
> My Redmi Go has a 1.3GB system partition...?

yes but wierd you can only install 950 mb gsi images did you try to install gsi 1gb above?

**Comment by phhusson** [comment]
I installed my latest releases, both oreo and pie, which have 1.3GB system image and it flashed and boots fine

**Comment by phhusson** [comment]
Can you run and dump `fastboot  getvar all` and `cat /proc/partitions`?

**Comment by proganime1200** [comment]
yes ill try bro i downloaded many images but only 950 images can be installed

**Comment by proganime1200** [comment]
(bootloader) version:0.5
(bootloader) battery-soc-ok:yes
(bootloader) battery-voltage:0
(bootloader) variant:QRD eMMC
(bootloader) unlocked:no
(bootloader) secure:yes
(bootloader) version-baseband:
(bootloader) version-bootloader:
(bootloader) DP:0x0
(bootloader) display-panel:
(bootloader) off-mode-charge:1
(bootloader) charger-screen-enabled:1
(bootloader) max-download-size: 0x1ff00000
(bootloader) partition-type:userdata:f2fs
(bootloader) partition-size:userdata:    0x14f1fbe00
(bootloader) partition-type:cache:ext4
(bootloader) partition-size:cache:       0x9600000
(bootloader) partition-type:system:ext4
(bootloader) partition-size:system:      0x52e00000
(bootloader) partition-type:vendor:
(bootloader) partition-size:vendor:      0x13e00000
(bootloader) partition-type:recovery:
(bootloader) partition-size:recovery:    0x1800000
(bootloader) partition-type:boot:
(bootloader) partition-size:boot:        0x3000000
(bootloader) partition-type:modem:
(bootloader) partition-size:modem:       0x5400000
(bootloader) partition-type:logdump:
(bootloader) partition-size:logdump:     0x1000000
(bootloader) partition-type:bk03:
(bootloader) partition-size:bk03:        0x40000
(bootloader) partition-type:dpo:
(bootloader) partition-size:dpo:         0x2000
(bootloader) partition-type:msadp:
(bootloader) partition-size:msadp:       0x40000
(bootloader) partition-type:apdp:
(bootloader) partition-size:apdp:        0x40000
(bootloader) partition-type:mcfg:
(bootloader) partition-size:mcfg:        0x400000
(bootloader) partition-type:syscfg:
(bootloader) partition-size:syscfg:      0x80000
(bootloader) partition-type:dip:
(bootloader) partition-size:dip:         0x100000
(bootloader) partition-type:mota:
(bootloader) partition-size:mota:        0x80000
(bootloader) partition-type:oem:
(bootloader) partition-size:oem:         0x100000
(bootloader) partition-type:config:
(bootloader) partition-size:config:      0x8000
(bootloader) partition-type:keystore:
(bootloader) partition-size:keystore:    0x80000
(bootloader) partition-type:misc:
(bootloader) partition-size:misc:        0x100000
(bootloader) partition-type:splash:
(bootloader) partition-size:splash:      0xb00000
(bootloader) partition-type:modemst2:
(bootloader) partition-size:modemst2:    0xa00000
(bootloader) partition-type:modemst1:
(bootloader) partition-size:modemst1:    0xa00000
(bootloader) partition-type:dsp:
(bootloader) partition-size:dsp:         0x1000000
(bootloader) partition-type:ssd:
(bootloader) partition-size:ssd:         0x2000
(bootloader) partition-type:fsc:
(bootloader) partition-size:fsc:         0x400
(bootloader) partition-type:persist:
(bootloader) partition-size:persist:     0x2000000
(bootloader) partition-type:fsg:
(bootloader) partition-size:fsg:         0xa00000
(bootloader) partition-type:sec:
(bootloader) partition-size:sec:         0x4000
(bootloader) partition-type:limits:
(bootloader) partition-size:limits:      0x8000
(bootloader) partition-type:devinfo:
(bootloader) partition-size:devinfo:     0x100000
(bootloader) partition-type:DDR:
(bootloader) partition-size:DDR:         0x8000
(bootloader) partition-type:abootbak:
(bootloader) partition-size:abootbak:    0x100000
(bootloader) partition-type:keymasterbak:
(bootloader) partition-size:keymasterbak:        0x100000
(bootloader) partition-type:cmnlib64bak:
(bootloader) partition-size:cmnlib64bak:         0x100000
(bootloader) partition-type:cmnlibbak:
(bootloader) partition-size:cmnlibbak:   0x100000
(bootloader) partition-type:aboot:
(bootloader) partition-size:aboot:       0x100000
(bootloader) partition-type:keymaster:
(bootloader) partition-size:keymaster:   0x100000
(bootloader) partition-type:cmnlib64:
(bootloader) partition-size:cmnlib64:    0x100000
(bootloader) partition-type:cmnlib:
(bootloader) partition-size:cmnlib:      0x100000
(bootloader) partition-type:devcfgbak:
(bootloader) partition-size:devcfgbak:   0x40000
(bootloader) partition-type:tzbak:
(bootloader) partition-size:tzbak:       0x200000
(bootloader) partition-type:rpmbak:
(bootloader) partition-size:rpmbak:      0x80000
(bootloader) partition-type:sbl1bak:
(bootloader) partition-size:sbl1bak:     0x80000
(bootloader) partition-type:devcfg:
(bootloader) partition-size:devcfg:      0x40000
(bootloader) partition-type:tz:
(bootloader) partition-size:tz:  0x200000
(bootloader) partition-type:rpm:
(bootloader) partition-size:rpm:         0x80000
(bootloader) partition-type:sbl1:
(bootloader) partition-size:sbl1:        0x80000
(bootloader) crc:1
(bootloader) serialno:6c4eaace7d85
(bootloader) kernel:lk
(bootloader) product:tiare
all: 6c4eaace
Finished. Total time: 0.394s

**Comment by phhusson** [comment]
So your system partition is indeed 1326MiB, so yeah, please test v113/v31
(previous versions were 1.5GiB so too big)


Le ven. 17 mai 2019 à 11:32, Ralph Garcia <notifications@github.com> a
écrit :

**Comment by proganime1200** [comment]
ok bro i will try

**Comment by proganime1200** [comment]
btw can you fix the google not certified permanent because everytime i fromat the phone it always  triggers not ccertified and hard to login i had to register the device id everytime

**Comment by proganime1200** [comment]
and btw can you add feature ota update so we can update easily bro

**Comment by proganime1200** [comment]
btw bro what gis build are you using for daily use that is not buggy one
